zerver/lib/url_encoding.py

```diff
@@ -58,6 +58,27 @@ def near_message_url(realm: Realm, message: Dict[str, Any]) -> str:
     return url
 
 
+def stream_topic_near_url(stream: Stream, topic: str, message: int) -> str:
```

Comment:
The `message_id` parameter should be named `message_id` -- `message` is a local variable name that we reserve for actual message objects.